lib/Kernel.php

```diff
@@ -155,6 +155,11 @@ public function registerContainerConfiguration(LoaderInterface $loader)
         if (file_exists($systemConfigFile)) {
             $loader->load($systemConfigFile);
         }
+
+        if (is_dir(PIMCORE_CONFIGURATION_DIRECTORY . '/imagethumbnails/')) {
```

Comment:
```suggestion
        if (is_dir(PIMCORE_CONFIGURATION_DIRECTORY . '/image-thumbnails/')) {
```
We're using `-` in other places as well - just to be consistent 😊